packaging/systemd/microshift-ovs-init.service

```diff
@@ -6,6 +6,6 @@ Requires=openvswitch.service NetworkManager.service
 
 [Service]
 Type=oneshot
-ExecStart=/bin/bash /usr/bin/configure-ovs.sh OVNKubernetes
+ExecStart=/bin/bash /usr/bin/configure-ovs.sh
```

Comment:
Where do we revert the previous changes applied by `configure-ovs.sh OVNKubernetes`?

Reply:
In /usr/bin/configure-ovs.sh, the NM key files will be removed.